test/atomex_test.exs

```diff
@@ -15,46 +19,47 @@ defmodule AtomexTest do
 
     Logger.debug("Full render of a simple feed:\n" <> result)
 
-    assert result == String.trim("""
-    <?xml version="1.0" encoding="UTF-8"?>
-    <feed xmlns="http://www.w3.org/2005/Atom">
-      <link href="https://example.com/feed" rel="self"/>
-      <author>
-        <name>David Gilmour</name>
-      </author>
-      <author>
-        <name>Steven Wilson</name>
-        <email>steven@example.com</email>
-      </author>
-      <id>https://example.com/</id>
-      <title>My incredible feed</title>
-      <updated>2016-05-24T13:26:08.003Z</updated>
-      <entry>
-        <summary>Hello World</summary>
-        <link href="https://example.com/entry/1" rel="alternate"/>
-        <content>Hello World, I'm a very long content</content>
-        <author>
-          <name>Andrew Latimer</name>
-          <uri>https://example.com/users/~Andrew</uri>
-        </author>
-        <id>https://example.com/entry/1</id>
-        <title>New entry: 1</title>
-        <updated>2016-05-24T12:26:08.003Z</updated>
-      </entry>
-      <entry>
-        <summary>Hello World</summary>
-        <link href="https://example.com/entry/2" rel="alternate"/>
-        <content>Hello World, I'm a very long content</content>
-        <author>
-          <name>Andrew Latimer</name>
-          <uri>https://example.com/users/~Andrew</uri>
-        </author>
-        <id>https://example.com/entry/2</id>
-        <title>New entry: 2</title>
-        <updated>2016-05-24T11:26:08.003Z</updated>
-      </entry>
-    </feed>
-    """)
+    assert result ==
+             String.trim("""
+             <?xml version="1.0" encoding="UTF-8"?>
+             <feed xmlns="http://www.w3.org/2005/Atom">
+               <link href="https://example.com/feed" rel="self"/>
+               <author>
+                 <name>David Gilmour</name>
+               </author>
+               <author>
+                 <name>Steven Wilson</name>
+                 <email>steven@example.com</email>
+               </author>
+               <id>https://example.com/</id>
+               <title>My incredible feed</title>
+               <updated>2016-05-24T13:26:08.003Z</updated>
+               <entry>
+                 <summary>Hello World</summary>
+                 <link href="https://example.com/entry/1" rel="alternate"/>
+                 <content>Hello World, I&apos;m a very long content</content>
```

Comment:
@Betree Sorry for not mentioning this earlier, but notice the `'` changing to `&apos;` which seems normal but could imply breaking changes.